src/js/index.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
```

Comment:
Why did you change this?

Reply:
It seems to me that it looks like better without empty line, but if it is important for you then i will return it.